inc/class-cachify.php

```diff
@@ -852,8 +823,8 @@ public static function pre_comment( $approved, $comment ) {
 	 * @param   string $old_status  Old status.
 	 * @param   object $comment     The comment.
 	 */
-	public static function touch_comment( $new_status, $old_status, $comment ) {
-		if ( $new_status !== $old_status ) {
+	public static function comment_status( $new_status, $old_status, $comment ) {
```

Comment:
I like the new method name, however we should either introduce an alias and deprecate the old or keep the name for now.

Reply:
Yes, let's deprecate this one as well.

Reply:
resolved in feab290d58cb32083c8fd2ae105930cab2dac7d3